spec/swagger_schemas/requests/participant_withdraw_request.rb

```diff
@@ -0,0 +1,40 @@
+PARTICIPANT_WITHDRAW_REQUEST = {
+  description: "A participant withdraw request",
+  type: :object,
+  properties: {
+    data: {
+      description: "A participant withdraw request data",
+      type: :object,
+      required: %i[type attributes],
+      properties: {
+        type: {
+          description: "The data typed",
+          type: :string,
+          example: "participant-withdraw",
+        },
+        attributes: {
+          description: "A participant withdraw request attributes",
+          type: :object,
```

Comment:
Think we need to add a `required: %i[reason course_identifier],` here so we have a red * next to the required fields in the schema div. Same for the others actions requests schemas.

![image](https://github.com/DFE-Digital/npq-registration/assets/3020581/00ba67f4-b9a6-4e6d-b546-e57f19f135aa)

Reply:
Good spot 👍 I'll update all the request schemas to be consistent; looks like you need `required` on both the parent attribute (specifying the children) and on the children (set to `true`) to get both the asterisk and `required: true` text.